server/remote_cache/hit_tracker/hit_tracker.go

```diff
@@ -463,20 +470,21 @@ func (t *TransferTimer) CloseWithBytesTransferred(bytesTransferredCache, bytesTr
 	if t.sizeCounter == UploadSizeBytes {
 		compressedSizeCounter = UploadTransferredSizeBytes
 	}
-	if err := h.c.IncrementCount(h.ctx, h.counterKey(), h.counterField(compressedSizeCounter), bytesTransferredClient); err != nil {
+	if err := h.c.IncrementCount(h.ctx, h.counterKey(), h.counterField(compressedSizeCounter), bytesTransferred); err != nil {
 		return err
 	}
-	if err := h.c.IncrementCount(h.ctx, h.counterKey(), h.counterField(t.timeCounter), dur.Microseconds()); err != nil {
+	if err := h.c.IncrementCount(h.ctx, h.counterKey(), h.counterField(t.timeCounter), duration.Microseconds()); err != nil {
 		return err
 	}
 	if *detailedStatsEnabled {
 		stats := &detailedStats{
 			Status:               t.actionCounter,
 			StartTime:            t.start,
-			Duration:             dur,
+			Duration:             duration,
 			Compressor:           compressor,
-			TransferredSizeBytes: bytesTransferredClient,
+			TransferredSizeBytes: bytesTransferred,
 		}
+		// TODO(iain): WTF does this do?
 		if err := h.recordDetailedStats(t.d, stats); err != nil {
```

Comment:
it stores the entries that show up in the "Cache requests" card under the cache tab

If bazel is hitting the cache proxy directly, then it means we're currently losing those scorecard entries.
Executor requests aren't shown in the cache requests card currently (too much data).